app/models.py

```diff
@@ -2763,6 +2774,31 @@ def render(template_name, **kwargs) -> str:
         )
 
 
+class Partner(Base, ModelMixin):
```

Comment:
i wonder if we can keep this class at the same location to reduce code diff?

Reply:
It has to be on top of SLDomain so the type resolver has seen it before it gets to SLDomain. Otherwise the static analizer gives an error.